### PR TITLE
reorder code in launcher.py

### PR DESCRIPTION
**Pull request by zeanne**
- sends message immediately after connecting to network.
- closes connection after message sent.
- renamed CloudyLauncher.py to cloudy_launcher.py

### CODE REVIEW

File: cloudy_launcher.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+                    print("\n")
+            
+            except socket.error as error:
+                print(os.strerror(error.errno));
+
+            finally:
+                cpsocket.close()
+
+
+        elif (playerInput.split(' ')[0] == QUIT):
```
Comment (bloodelves88) [inline cloudy_launcher.py]: Standardize to `playerInput.split()`.

File: cloudy_launcher.py [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+            except socket.error as error:
+                print(os.strerror(error.errno));
+
+            finally:
+                cpsocket.close()
+
+
+        else:
+
```
Comment (bloodelves88) [inline cloudy_launcher.py]: Remove this blank line while you're at it

File: CloudyLauncher.py [hunk 3]
```diff
@@ -1,95 +0,0 @@
...
-                response = cpsocket.recv(BUFFER_SIZE)
-                response = response.decode("utf-8")
-
-                if (response.isdigit()):
-                    print("call thin client with controllerid '" + str(response) + "'\n")
-                    thin_client.startClient(int(response))
-
-                else:
-                    print(response)
-                    print("\n")
```
Comment (9gix) [inline CloudyLauncher.py]: if you want new line, use `print()` instead

File: cloudy_launcher.py [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+    
+    while (True):
+
+        playerInput = input("Enter command: ")
+        print(playerInput)
+
+        if (playerInput == EXIT):
+            break
+
+        elif ((len(playerInput.split()) == 0) | (len(playerInput.split()) != 2)):
```
Comment (bloodelves88) [inline cloudy_launcher.py]: You don't need this condition: `(len(playerInput.split()) == 0)`

Also don't use bitwise operations for boolean expressions.

File: cloudy_launcher.py [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+                response = response.decode("utf-8")
+                print(response)
+
+                if (response.isdigit()):
+                    print("open process to launch thin client with controllerid '" + str(response) + "'\n")
+                    thin_client.startClient(int(response))
+
+                else:
+                    print(response)
+                    print("\n")
```
Comment (9gix) [inline cloudy_launcher.py]: use `print()` instead

File: cloudy_launcher.py [hunk 6]
```diff
@@ -0,0 +1,110 @@
...
+import thin_client
+
+
+JOIN = "join"
+QUIT = "quit"
+EXIT = "exit"
+COMMANDS = "Commands: \n" + \
+           "connect and join user -- join <username> \n" + \
+           "connect and quit user -- quit <username> \n" + \
+           "exit cloudy_launcher  -- exit \n"
```
Comment (9gix) [inline cloudy_launcher.py]: consider multi-line string instead:

    COMMANDS  = """
    """

File: cloudy_launcher.py [hunk 7]
```diff
@@ -0,0 +1,110 @@
...
+
+
+if __name__ == "__main__":
+
+    ## to connect with CloudyPanel
+    TCP_IP = "127.0.0.1"
```
Comment (9gix) [inline cloudy_launcher.py]: wrap everything inside a function. thus you just call:

    if __name__ == "__main__":
        main()

File: cloudy_launcher.py [hunk 8]
```diff
@@ -0,0 +1,110 @@
...
+JOIN = "join"
+QUIT = "quit"
+EXIT = "exit"
+COMMANDS = "Commands: \n" + \
+           "connect and join user -- join <username> \n" + \
+           "connect and quit user -- quit <username> \n" + \
+           "exit cloudy_launcher  -- exit \n"
+
+
+if __name__ == "__main__":
```
Comment (9gix) [inline cloudy_launcher.py]: wrap every logic inside a function. thus you just call:

    if __name__ == "__main__":
        main()

File: cloudy_launcher.py [hunk 9]
```diff
@@ -0,0 +1,110 @@
...
+        print(playerInput)
+
+        if (playerInput == EXIT):
+            break
+
+        elif ((len(playerInput.split()) == 0) | (len(playerInput.split()) != 2)):
+            print("Invalid command.\n")
+            print(COMMANDS)
+
+        elif (playerInput.split()[0] == JOIN):
```
Comment (9gix) [inline cloudy_launcher.py]: refactor all of these independent `split()` into `inputParams = playerInput.split()` as a  variable

Comment (9gix) [inline cloudy_launcher.py]: can be further refactor, using "command-action" mapping.

    def join():
        # Do whatever necessary
        pass

    cmd_mapping = {
        JOIN: join(),
        QUIT: quit(),
        # etc...
    }
    try:
        cmd_mapping[player_cmd]()
    except KeyError:
        # handle Command Not Found
        print("Command Not found")

**Comment by 9gix** [comment]
Looks like there are new line character differences between @bloodelves88 & @zeanne 
Because all of the source code lines was changed.

File: cloudy_launcher.py [hunk 10]
```diff
@@ -0,0 +1,120 @@
...
+        cpsocket.close()
+
+def exitLauncher(playerInput):
+    quit()
+
+cmd_mapping = {
+JOIN: joinGame,
+QUIT: quitGame,
+EXIT: exitLauncher
+}
```
Comment (bloodelves88) [inline cloudy_launcher.py]: Indent:
```python
cmd_mapping = {
    JOIN: joinGame,
    QUIT: quitGame,
    EXIT: exitLauncher
}
```

**Pull request merged**